### PR TITLE
2255 lesmis login flow

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add login page and form
- Add router and routes
- Add react query to connect to api
- Add placeholder view templates

---

### Screenshots:
See issue

### CODE REVIEW

File: packages/lesmis/src/components/Layout.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import styled from 'styled-components';
+import MuiContainer from '@material-ui/core/Container';
+import MuiBox from '@material-ui/core/Box';
+
+const desktopWidth = '768px';
+
+/*
+ * Wrapping container
+ */
+export const Container = styled(MuiContainer)`
```
Comment (tcaiger) [inline packages/lesmis/src/components/Layout.js:16]: No need to review this file. The layout is coming later on card: https://github.com/beyondessential/tupaia-backlog/issues/2279

File: packages/lesmis/src/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+import { FlexSpaceBetween } from './Layout';
+import { ProfileButton } from './ProfileButton';
+import { NAVBAR_HEIGHT } from '../constants';
+
+/**
+ * === PLACEHOLDER NAVIGATION ===
+ *
+ * Waiting for designs to be completed. @see https://github.com/beyondessential/tupaia-backlog/issues/2234
+ *
```
Comment (tcaiger) [inline packages/lesmis/src/components/NavBar.js:20]: No need to review here. There is another PR for the navbar

File: packages/lesmis/src/views/DistrictView.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
```
Comment (tcaiger) [inline packages/lesmis/src/views/DistrictView.js:5]: No need to review here. this is just a placeholder template

File: packages/lesmis/src/views/HomeView.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
```
Comment (tcaiger) [inline packages/lesmis/src/views/HomeView.js:4]: No need to review here. this is just a placeholder template

File: packages/lesmis/src/views/ProfileView.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
```
Comment (tcaiger) [inline packages/lesmis/src/views/ProfileView.js:7]: No need to review here. this is just a placeholder template

File: packages/lesmis/src/views/ProvinceView.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '../components';
```
Comment (tcaiger) [inline packages/lesmis/src/views/ProvinceView.js:7]: No need to review here. this is just a placeholder template

File: packages/lesmis/src/views/SchoolView.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '../components';
```
Comment (tcaiger) [inline packages/lesmis/src/views/SchoolView.js:7]: No need to review here. this is just a placeholder template

**Review by EMcQ-BES — APPROVED** [review]
This was interesting to look through and study up a bit of react

**Pull request merged**